content/3_Silver/Flood_Fill.mdx

```diff
@@ -554,7 +556,7 @@ Recursive implementations of flood fill sometimes lead to
 
 </Warning>
 
-A non-recursive implementation of flood fill follows:
+A non-recursive implementation of flood fill adds the edges to a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:
```

Comment:
I'm pretty sure you add nodes, not edges to the stack/queue in floodfill. This is also the case in the code below.
```suggestion
A non-recursive implementation of flood fill adds adjacent nodes to a stack or queue, similar to [BFS](https://usaco.guide/gold/bfs), and is usually implemented as follows:
```